Makefile

```diff
@@ -0,0 +1,10 @@
+install: ##do all the npm commands to run the project
+	npm install
+	npm run build
+	upm link
+
+test: #run test suite 
+	npm test
+
+test_example: ##run test using the manifest file
```

Comment:
Need to add `cp sample-input.yml test.yml`, but looks good otherwise!

Reply:
I wasn't totally sure that people would want to cp over the example file every time they want to run it (eg: if they have made some changes in the sample file they're using)

Reply:
(But I do agree if you haven't already copied it over it would be a problem)

Reply:
Is there a way to check that one already exists, and only copy it if there isn't one?

Reply:
Yeah I can do that